worlds/alttp/ItemPool.py

```diff
@@ -798,7 +837,9 @@ def place_item(loc, item):
         pool.extend(['Moon Pearl'] * customitemarray[28])
 
     if world.smallkey_shuffle[player] == smallkey_shuffle.option_universal:
-        itemtotal = itemtotal - 28  # Corrects for small keys not being in item pool in universal mode
+        itemtotal = itemtotal - 28  # Corrects for small keys not being in item pool in Retro Mode
```

Comment:
```suggestion
        itemtotal = itemtotal - 28  # Corrects for small keys not being in item pool in universal Mode
```
universal is correct, there is no retro option in AP. Closest is Retro Bow.

Reply:
Probably a merge conflict issue where I accidentally reverted a change there